extensions/wikia/SiteWideMessages/SpecialSiteWideMessages_body.php

```diff
@@ -1031,7 +1031,7 @@ static function getAllUserMessagesId($user, $filter_seen = true) {
 			. ' WHERE msg_removed = ' . MSG_REMOVED_NO
 			. ' AND msg_mode = ' . MSG_MODE_ALL
 			. ' AND (msg_expire IS NULL OR msg_expire > ' . $DB->AddQuotes(date('Y-m-d H:i:s')) . ')'
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal " AND (msg_expire IS NULL OR msg_expire > " 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)